views/ipv/page/page-multiple-doc-check.njk

```diff
@@ -41,6 +42,16 @@
 %}}
 
 {% block content %}
+  {% if displayOutageBanner %}
+    {% set html %}
+      <h3 class="govuk-notification-banner__heading">{{ 'pages.pageMultipleDocCheck.content.outageBanner.title' | translate }}</h3>
+      <p class="govuk-body">{{ 'pages.pageMultipleDocCheck.content.outageBanner.paragraph1' | translate }}</p>
+      <p class="govuk-body">{{ 'pages.pageMultipleDocCheck.content.outageBanner.paragraph2' | translate }} <a class="govuk-link" href="/ipv/page/pyi-post-office">{{ 'pages.pageMultipleDocCheck.content.outageBanner.linkText' | translate }}</a>.</p>
```

Comment:
I don't think this link is quite right - it'll take you to the page, but we need to go via the journey engine otherwise the session won't be updated properly.

You could try `/ipv/journey/page-multiple-doc-check/end`